Sources_v3/Controllers/MessageController/MessageController.swift

```diff
@@ -147,6 +147,38 @@ public extension _ChatMessageController {
             }
         }
     }
+    
+    /// Creates a new reply message locally and schedules it for send.
+    ///
+    /// - Parameters:
+    ///   - text: Text of the message.
+    ///   - showReplyInChannel: Set this flag to `true` if you want the message to be also visible in the channel, not only
+    ///   in the response thread.
+    ///   - extraData: Additional extra data of the message object.
+    ///   - completion: Called when saving the message to the local DB finishes.
+    ///
+    func reply(
```

Comment:
I like the simplicity of `reply` but maybe we should be consistent with `ChannelController` and use `createNewReply` instead?

Reply:
Maybe 🤷‍♂️
I also liked the simplicity 
Maybe instead we can do `createNewMessage` -> `message`?

Reply:
We have `controller.messages` I don't think `controller.message()` is a good idea. The name also implies that the message is created not sent - the same is true for replies. I'd follow the pattern from channel controller tbh.